compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/resourcepath/util/PathSegment.java

```diff
@@ -45,6 +47,10 @@ public interface PathSegment {
      * @since 2.0.0
      */
     enum Kind {
+        /**
+         * Represents the rooth path segment in the resource path which is just ".".
+         */
+        ROOT_PATH,
```

Comment:
Isn't this [dot-resource-path](https://ballerina.io/spec/lang/master/#dot-resource-path)? Root path is something else?

Reply:
We don't need this path-segment kind. This is corrected via https://github.com/ballerina-platform/ballerina-lang/pull/39477/commits/fc6c7f51748d5f7d186efc3603328691f70f76b4